src/app/clients/ClientsPage.tsx

```diff
@@ -48,24 +48,25 @@ const collectionCentreToAbbreviation = (
     }
 };
 
-export const clientTableHeaderKeysAndLabels: TableHeaders = [
-    ["iconsColumn", ""],
+export const clientTableHeaderKeysAndLabels: TableHeaders<ClientsTableRow> = [
     ["fullName", "Name"],
     ["familyCategory", "Family"],
     ["addressPostcode", "Postcode"],
     ["deliveryCollection", ""],
-    ["packingDate", "Packing Date"],
+    ["packingDatetime", "Packing Date"],
```

Comment:
Why do we have packingDateTime if we also have packingTimeLabel?

Reply:
`packingDatetime` is rendered as just a date, but keeping the data intact means sorting works. The `packingLabel` column just renders as AM or PM. In these tables it doesn't really matter if there's data duplication as this should just be used for rendering and sorting logic. If you want a way to trace back to the original data, just add a primary key to it.